### ISSUE TITLE
Active Visual Studio configuration not considered when running tasks

### ISSUE THREAD

**Issue by hummigbird1** [nuke-build/nuke#517]
So I might be missing something obvious here but I couldn't find anything about it.
(BTW: The lack of useful help and documentation might be a real dealbreaker for this great build system IMHO)

But back to the issue at hand:
I was just getting into setting up my project with Nuke and suddenly noticed that regardless of what Configuration I have selected in Visual Studio the Nuke build always uses Debug configuration.
I am aware that this is the default for "IsLocalBuild" (and here the next question I could not find any answers to , what defines "IsLocalBuild" anyways?) 
Now I am aware that this might be related to the Visual Studio Extension but as I could not find any way to file an issue in this repository ... 

I have played around with Precompiler constants (#if DEBUG) but that also seems to have little to no effect at all, which right now I am not actually understanding. (Might have to do with the fact that the _build.csproj is never built somehow according to the Solution configuration)  

**So in short:** 
Running a Task from the Visual Studio Extension always runs the Debug Configuration and I see no way how to change that.
I can not imagine that I am the only one with that problem so I am missing something obvious here, please enlighten me!

**Comment by matkoch** [comment]
Not a good way to start a conversation here, especially given the issue template says that starring the project is a requiremet to create issues. I also wonder what that _deal_ is that's gonna break?!

**Comment by hummigbird1** [comment]
My apologies. I didn't mean to sound rude.
I assume you know how it is when you search and find no place to start and the frustration starts to rise because it so obvious you are missing something but can't for the life of you, figure out what.

And to be honest, I didn't actually understand what "starring the project" meant. 
But since you mentioned it now I have figured it out, now. As I a said, frustrations made it hard to think straight.
Admittedly not the best place to post a "issue" ... sorry. 

So the dealbreaker I was mentioning is that it seems near impossible to use a build system with so much possibilities without having any kind of reference or extensive documentation (beyond the "getting started" part, that is). 
Again, I just might be inept at googling stuff so any pointers would be much appreciated.

So any chance you could point me at what I am overlooking here?

**Comment by matkoch** [comment]
I guess you shouldn't treat open-source projects as some kind of black box, that you can handle just by looking at the documentation. Finding an answer what defines `IsServerBuild` and `IsLocalBuild` seems fairly easy:

https://github.com/nuke-build/nuke/search?q=IsServerBuild&unscoped_q=IsServerBuild

> Without any kind of reference

I'm not sure what your measurement is, but the documentation is there and has proven helpful for others in many situations already.

And I meant to say that the word _deal_ here is a bit inappropriate. Usually, a deal is something that both parties can benefit from... You get a framework for free (3 years of work)... you would also get help for free (in most cases)... but I won't feel like helping if the first thing coming here is complaining. Also, your frustration has absolutely no relevance here.
